financials/income_log.csv

```diff
@@ -18,4 +18,113 @@ income_id,date,type,source,currency,amount,exchange_rate,USD_equivalent,to_addre
 23,20190411,donation,Sparkpool,GRIN,3490.0,2.87,10016.30,https://donations.grin-tech.org,-,Sparkpool & anonymous miner donation,
 24,20190413,conversion,lehnberg,BTC,0.12263024,5090.29,624.22,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,369e9a17844d98be7addeca75dee36e1f78b644e6863fc664b0bda90fadbc565,Conversion of ETH to BTC,
 25,20190315,donation,Poloniex,BTC,1.27864,3960.91,5064.58,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,007815dfcf6ad13016e6b190c49858456b94e0b21c5f3f49fc2fa61c8b929443,Monthly donation to general fund #1,https://medium.com/circle-blog/poloniex-enables-grin-withdrawals-donates-to-grin-general-fund-114906884f95
-26,20190416,donation,Poloniex,BTC,0.67078,5235.56,3511.91,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,1cd64e2fcebb0e57057887737d38e1853bbcf9de7eafadd9d8e75c51f0e6b7b7,Monthly donation to general fund #2,
\ No newline at end of file
+26,20190416,donation,Poloniex,BTC,0.67078,5235.56,3511.91,3ChVP627KU5w4zu2rieFPF3wGXWQgmhvrs,1cd64e2fcebb0e57057887737d38e1853bbcf9de7eafadd9d8e75c51f0e6b7b7,Monthly donation to general fund #2,
+27,20190301,change,N/A,BTC,16.30469134,3859.58,62929.26,3JAqRiaWgFvARgpDL31eZ696Dt59nFUYjw,022feacd15899425ee5a164f6e67a3f8701ce4fecbad8bf4532f2576f172a468,change as part of paying coinspect deposit,

```

Comment:
Does it make sense to track change? You would only log the spent amount in the spending log and that should be enough?

Reply:
I'd prefer to avoid as well, but it creates inconsistencies if we do not, and it makes our reporting opaque. See the actual transaction in question: https://www.blockchain.com/btc/tx/022feacd15899425ee5a164f6e67a3f8701ce4fecbad8bf4532f2576f172a468

If we only track the 4.482 BTC payment to coinspect, but _not_ the change output, it might as well be that we siphoned 16.30469134 BTC off somewhere else. For someone in the community wanting to audit the project's funds and spends, it would be hard to tell the difference. This amount is now also in a new wallet address, `3JAqRiaWgFvARgpDL31eZ696Dt59nFUYjw`, and that should arguably also be tracked and logged properly.

Happy to consider any better suggestions, this felt to me as the most straight forward and simple thing to do for the time being.